src/i10_bluesky/plans/align_slits.py

```diff
@@ -118,7 +116,7 @@ def align_s5s6(
     """
 
     if det is None:
-        det, det_name = get_rasor_default_det()
+        det, _ = get_rasor_default_det()
```

Comment:
Should: It looks like you're no longer using `det_name`? So you could just not return it from this function now?